### ISSUE TITLE
Explore automated detection of which metadata files are present in a product repo

### ISSUE THREAD

**Issue by maherou** [E4S-Project/E4S-Documenter#2]
The e4s_product.yaml file presently lists which files are present in each product repo.  I think it would be better to discover which files were present by looking for them as part of the raking process.  This way, product teams could add a new file and it would be discovered automatically.  

In general, I think it would be better to have no information other than the base URL in the e4s_product.yaml file. 

Perhaps you are already moving in this direction, but want to make sure.

**Comment by wspear** [comment]
Yes, I just pulled in the prototype to work from. My current plan is to reduce the e4s_product.yaml file to just the project names and URL's (and possibly category info for sorting) and get other information from an e4s.yaml file either provided by the project or, as a backup, provided in project-specific e4s.yaml files from our repo here.

**Comment by maherou** [comment]
In the spirit of allowing teams to manage all content in their repositories, I think it would be best to have __all__ information in the individual product repos.  We should only need the URL.  Even the product name and category is best raked (I think) from the repo.

**Comment by wspear** [comment]
That seems workable. I think I will end up using the repo name as seen in the URL for some purposes then, at least to keep local/cached project metadata in human-readable locations.
